app/models.py

```diff
@@ -1062,11 +1076,16 @@ class Alias(db.Model, ModelMixin):
 
     # have I been pwned
     hibp_last_check = db.Column(ArrowType, default=None)
-    hibp_breaches = db.relationship("Hibp", secondary="alias_hibp")
+    hibp_breaches_notified_user = db.relationship("Hibp", secondary="alias_hibp")
```

Comment:
Oh I didn't know we can share the same secondary class for a many-to-many fields!